libmachine/log/bugsnag_test.go

```diff
@@ -0,0 +1,11 @@
+package log
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestBugSnag(t *testing.T) {
```

Comment:
Yes, that more a test that help building and testing quickly, not sure what it will become